appveyor.yml

```diff
@@ -0,0 +1,22 @@
+version: 0.8.{build}
```

Comment:
As discussed, we should change this file to .appveyor.yml.  -- also we need to think about version numbering handling here.